run_nextflow_dims.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+set -euo pipefail
+
+workflow_path='/hpc/dbg_mz/development/DIMS_Nextflow'
```

Comment:
is er een productie map waarnaar verwezen kan worden, of draait DIMS_Nextflow alleen onder developement?

Reply:
De Nextflow pipeline runt nog niet in productie. workflow_path moet aangepast worden naar pad voor productie versie na validatie.